web/js/containers/map-interactions.js

```diff
@@ -73,13 +73,20 @@ export class MapInteractions extends React.Component {
   }
 
   render() {
-    const { isShowingClick, mouseEvents } = this.props;
+    const {
+      isDistractionFreeModeActive,
+      isShowingClick,
+      mouseEvents
+    } = this.props;
     const mapClasses = isShowingClick ? 'wv-map' + ' cursor-pointer' : 'wv-map';
 
     return (
       <React.Fragment>
         <div id="wv-map" className={mapClasses} />
-        <OlCoordinates mouseEvents={mouseEvents} />
+        <OlCoordinates
```

Comment:
Probably could just prevent `ol-coords` from being rendered if in distactionFree to prevent passing prob and logic inside child